src/Filter.php

```diff
@@ -260,7 +260,7 @@ protected function performEqualityMatch($value, $rowValue, $ignoreCase = false)
             $rowValue = strtolower($rowValue);
             $value = is_array($value)
                 ? array_map('strtolower', $value)
-                : strtolower($value);
```

Comment:
if this is a problem, then the array_map call above as well

Reply:
I don't follow what you mean! Why should the call to `array_map` be a problem?

Reply:
I suppose phpstan complained about $value not being strictly a string but strtolower expects one. Since strtolower is also used in the map callback and entries in the array might also not all be strings, the same problem would occur if there's not a string inside